Formula/klaytn.rb

```diff
@@ -0,0 +1,39 @@
+class Klaytn < Formula
+  desc "Klaytn Blockchain Platform"
+  homepage "https://github.com/klaytn/klaytn"
+  url "https://github.com/klaytn/klaytn.git", :tag => "v1.1.1"
+
+  head "https://github.com/klaytn/klaytn.git", :branch => "master"
+
+  depends_on "go" => :build
+  depends_on :macos => :el_capitan
+
+  def install
+    system "go", "env"
+    system "make", "kcn" # if this fails, try separate make/make install steps
+    bin.install "build/bin/kcn"
+    
+    system "make", "kpn"
+    bin.install "build/bin/kpn"
+    
+    system "make", "ken"
+    bin.install "build/bin/ken"
+    
+    system "make", "kscn"
```

Comment:
I believe klaytn has other binaries for service chain, such as `kspn` and `ksen`. Why don't you also install them?

Reply:
Thank you for the notice. I listed all binaries generated by Klaytn.